components/infra-proxy-service/migrations/pipeline/phasetwomigration.go

```diff
@@ -73,19 +73,43 @@ func createProject(result <-chan PipelineData) <-chan PipelineData {
 }
 
 // PopulateUsers returns PhaseTwoPipelineProcessor
-func PopulateUsers() PhaseTwoPipelineProcessor {
+func PopulateUsers(service *service.Service) PhaseTwoPipelineProcessor {
 	return func(result <-chan PipelineData) <-chan PipelineData {
-		return populateUsers(result)
+		return populateUsers(result, service)
 	}
 }
 
-func populateUsers(result <-chan PipelineData) <-chan PipelineData {
+func populateUsers(result <-chan PipelineData, service *service.Service) <-chan PipelineData {
 	log.Info("Starting PopulateUsers routine")
 	out := make(chan PipelineData, 100)
 
 	go func() {
 		for res := range result {
-			log.Info("Processing to populateUsers...")
+			// Start user migration
+			_, err := service.Migration.StartUserMigration(res.Ctx, res.Result.Meta.MigrationID, res.Result.Meta.ServerID)
+			if err != nil {
+				log.Errorf("Failed to update `StartUserMigration` status in DB: %s :%s", res.Result.Meta.MigrationID, err)
+				return
```

Comment:
I am concerned that in case if we have failures in these cases, will the caller function wait since we are not sending any data back. Please check this.